HW_1/Task_2.py

```diff
@@ -0,0 +1,6 @@
+n = abs(int(input('enter three-digit number: ')))
+if (n) < 100 or (n) > 999:
+    print('it,s not a three-digit number, please enter again')
+else:
+    sum = (n % 10 + (n // 10) % 10 + n // 100)
```

Comment:
У операций деления одинаковый приоритет, поэтому скобки не нужны.
```suggestion
    sum = n % 10 + n // 10 % 10 + n // 100
```